kivy/uix/label.py

```diff
@@ -707,13 +710,23 @@ def on_ref_press(self, ref):
     .. versionchanged:: 1.9.0
         `padding_y` has been fixed to work as expected.
         In the past, the text was padded by the negative of its values.
+
+    .. deprecated:: 2.2.0
+        Please use :attr:`padding` instead.
     '''
```

Comment:
I may agree with the deprecation, but we can't remove the support for `padding_x` and `padding_y`, without giving the users a quite large deprecation window.

Is fine to mark `padding_x` and `padding_y` as deprecated, but we should keep the functionality for at least ~3/4 releases, so users have time to migrate their code at their own pace.

Reply:
Understood! Sounds like a good approach!